PE/__init__.py

```diff
@@ -757,17 +764,51 @@ def parseImports(self):
         if x is None:
             return
 
+        self.parseImportTable(x, irva, self.imports, is_imports=True)
+
+    def parseDelayImports(self):
+        self.delayImports = []
+
+        didir = self.getDataDirectory(IMAGE_DIRECTORY_ENTRY_DELAY_IMPORT)
+
+        # RP BUG FIX - invalid IAT entry will point of range of file
+        irva = didir.VirtualAddress
+        x = self.readStructAtRva(irva, 'pe.IMAGE_DELAY_IMPORT_DIRECTORY', check=True)
+        if x is None:
+            return
+
+        self.parseImportTable(x, irva, self.delay_imports, is_imports=False)
```

Comment:
So in `getDelayImports` and earlier in this function you call the list `self.delayImports` and here you call it `self.delay_imports`, which I think is the cause of the CI pipeline failures.

Reply:
That's a bad typo, sorry.  Fixed it.